### PR TITLE
more verbose errors during hash check, add decoding logging steps

### PR DESCRIPTION
**Pull request by Simontwice**
# Description

# Checklist
   - [x] Is this PR focused on a single feature without the possibility of being divided into smaller PRs?
   - [x] Are all variables set with logical defaults that are backward compatible?
   - [x] I attached link to neptune if it was necessary
   - [x] Have you successfully executed the linter and tests?
   - [x] Are all functions concise and don't require splitting?
   - [x] Is there documentation for arguments and complex logic?
   - [x] Are there no temporary solutions in the code? Should a task be added to Trello to refine the code?
   - [x] Are all functions placed in appropriate files?
   - [x] Is the PR name meaningful and descriptive?
   - [x] Is PR description provided, or unnecessary?
  
Other questions:
   - [x] If I made changes to the requirements or anything related to the Singularity image I did generate new image and upload a new image to the clusters
   - [x] If this change is significant enough I notified all individuals working with this repository (@channel in slack channel)
   - [x] If there is a need for a second reviewer I requested additional review

### CODE REVIEW

File: lizrd/support/sync_and_version.py
Comment (kuba-krj) [inline lizrd/support/sync_and_version.py:79]: shouldn't it be `d[user] if user in d else user[3:]`?

**Review by kuba-krj — CHANGES_REQUESTED** [review]
Contrary to the checklist, this PR is in fact **not** focused on one single feature. Please move adding `decoding_logging_steps` to a separate PR, and I can approve this part right away. Regarding the changes in `rsync`, there is a need for a second review from someone with a better technical understanding

**Comment by Simontwice** [comment]
> Contrary to the checklist, this PR is in fact **not** focused on one single feature. Please move adding `decoding_logging_steps` to a separate PR, and I can approve this part right away. Regarding the changes in `rsync`, there is a need for a second review from someone with a better technical understanding

very good review. I have since split some of the features to separate PRs.

**Pull request closed without merging**
